### PR TITLE
Show cluster ID with 'swarm info' if set

### PR DESCRIPTION
**Pull request by marians**
Related: https://github.com/giantswarm/cli/issues/436

The purpose of this change is to simplify debugging for users who have access to multiple clusters. It adds the `Cluster ID` output line if a cluster ID is set on the client side (as through the environment variable `SWARM_CLUSTER_ID`).

No cluster set:

```
Swarm CLI version:    0.23.0+git
API endpoint:         https://api.giantswarm.io/v1
API status:           reachable
Logged in as user:    marian
Current environment:  marian/dev
```

Cluster ID set:

```
Swarm CLI version:    0.23.0+git
API endpoint:         https://api.giantswarm.io/v1
API status:           reachable
Cluster ID:           aws-cluster-01.giantswarm.io
Logged in as user:    marian
Current environment:  giantswarm/production
```

### CODE REVIEW

**Comment by marians** [comment]
RFR @ewoutp @zyndiecate

**Comment by zyndiecate** [comment]
Is the position of the cluster ID on purpose?

File: cli/cli.go [hunk 1]
```diff
@@ -9,8 +9,8 @@ import (
 )
 
 const (
-	CLUSTER_PATTERN = "https://%s.giantswarm.io/%s"
-	CONFIG_DIR      = ".swarm"
+	ENDPOINT_PATTERN = "https://%s.giantswarm.io/%s"
```
Comment (zyndiecate) [inline cli/cli.go]: The capitalized-underscore constants are legacy. Go convention is to write them in upper-camel-case. So please change `ENDPOINT_PATTERN` to `EndpointPattern`.

**Comment by marians** [comment]
> Is the position of the cluster ID on purpose?

Yes, it is.

**Comment by marians** [comment]
Changed that const to `EndpointPattern`.

**Comment by marians** [comment]
@zyndiecate LGTY?

**Comment by zyndiecate** [comment]
Please make the new constant private for the package in case it is not required from outside the package. Then LGTM.

**Comment by marians** [comment]
@zyndiecate You mean `endpointPattern` instead of `EndpointPattern`?

**Comment by marians** [comment]
Changed to `endpointPattern`. Merging now.

**Pull request merged**